metadata-db-app/src/cmr/metadata_db/data/oracle/concepts.clj

```diff
@@ -431,18 +444,23 @@
                                                                 (where `(and (= :c.concept-id :t.concept-id)
                                                                              (= :c.revision-id :t.revision-id)))))]
                                      (doall (map (partial db-result->concept-map concept-type conn provider-id)
-                                                 (su/query conn stmt)))))))]
-      (when (not (= "efs-off" (efs-config/efs-toggle)))
-        (info "Runtime of EFS get-concepts: " (first efs-concepts-get) " ms.\nValues gotten from EFS: " (pr-str (second efs-concepts-get))))
-      (when (not (= "efs-only" (efs-config/efs-toggle)))
+                                                 (su/query conn stmt)))))))
+          dynamo-concepts-get (when (not= "dynamo-off" (dynamo-config/dynamo-toggle))
+                                (util/time-execution
+                                 ()))]
+      (when (not= "dynamo-off" (dynamo-config/dynamo-toggle))
+        (info "Runtime of EFS get-concepts: " (first efs-concepts-get) " ms."))
+      (when (not= "dynamo-only" (dynamo-config/dynamo-toggle))
         (info "Runtime of Oracle get-concepts: " (first oracle-concepts-get) " ms."))
+      (when (not= "dynamo-off" (dynamo-config/dynamo-toggle))
+        (info "Runtime of DynamoDB get-concepts: " (first dynamo-concepts-get) " ms."))
```

Comment:
same comment here - the when clauses on 456 and 451 are the same, should they be or can we combine the messages into 1 when?